web/ui/dashboard-component/projects/convoy-dashboard/src/lib/convoy-dashboard.service.ts

```diff
@@ -7,7 +7,7 @@ import { HTTP_RESPONSE } from './models/http.model';
 })
 export class ConvoyDashboardService {
 	token: string = '';
-	authType: 'Bearer' | 'Basic' = 'Basic';
+	authType!: 'Bearer' | 'Basic';
```

Comment:
Please why did you change to this @Oluwadaminiola

Reply:
this was causing an error on SAAS as authType kept returning BASIC, so I decided to rely on the values coming from the component itself